.travis.yml

```diff
@@ -1,31 +1,24 @@
-language: android
+os: osx
 
-jdk:
-  - openjdk8
-
-android:
-  components:
-    - tools
-    - platform-tools
-    - build-tools-28.0.3
-    - android-28
-    - android-22 # needed for the emulator
-    - sys-img-armeabi-v7a-android-22 # as of now, travis doesn't support x86 so we stick with armeabi
-  licenses:
-    - android-sdk-preview-license-.+
-    - android-sdk-license-.+
-    - google-gdk-license-.+
+osx_image: xcode11.3
 
 script:
+  - curl -s "https://get.sdkman.io" | bash
+  - source "$HOME/.sdkman/bin/sdkman-init.sh"
+  - sdk install java 8.0.242.hs-adpt
+  - curl -s https://dl.google.com/android/repository/commandlinetools-mac-6200805_latest.zip -o tools.zip
+  - export ANDROID_SDK_ROOT=~/.android
+  - mkdir $ANDROID_SDK_ROOT
+  - unzip tools.zip -d $ANDROID_SDK_ROOT
+  - export PATH=$ANDROID_SDK_ROOT/tools/bin:$PATH
+  - avdmanager --list
+  - yes | sdkmanager --licenses
```

Comment:
Here is what I have in one of my projects. If you put the license, Gradle will automatically install everything necessary 

https://github.com/tasomaniac/OpenLinkWith/blob/master/.travis.yml#L9

Reply:
I see that you also need to install system images for emulator